DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/README.md

```diff
@@ -183,14 +183,13 @@ PASSED
 | `CompareGzip.cpp`            | Contains code to compare a GZIP-compatible file with the original input.
 | `WriteGzip.cpp`              | Contains code to write a GZIP compatible file. 
 | `crc32.cpp`                  | Contains code to calculate a 32-bit CRC compatible with the GZIP file format and to combine multiple 32-bit CRC values. It is only used to account for the CRC of the last few bytes in the file, which are not processed by the accelerated CRC kernel. 
-| `kernels.hpp`                  | Contains miscellaneous defines and structure definitions required by the LZReduction and Static Huffman kernels.
-| `crc32.hpp`                    | Header file for `crc32.cpp`.
-| `gzipkernel.hpp`              | Header file for `gzipkernels.cpp`.
-| `gzipkernel)ll.hpp`              | Header file for `gzipkernels_ll.cpp`.
-| `CompareGzip.hpp`              | Header file for `CompareGzip.cpp`.
-| `pipe_array.hpp`                | Header file containing the definition of an array of pipes. 
-| `pipe_array_internal.hpp`       | Helper for pipe_array.hpp. 
-| `WriteGzip.hpp`                | Header file for `WriteGzip.cpp`. 
+| `kernels.hpp`                | Contains miscellaneous defines and structure definitions required by the LZReduction and Static Huffman kernels.
+| `crc32.hpp`                  | Header file for `crc32.cpp`.
+| `gzipkernel.hpp`             | Header file for `gzipkernels.cpp`.
+| `gzipkernel)ll.hpp`          | Header file for `gzipkernels_ll.cpp`.
+| `CompareGzip.hpp`            | Header file for `CompareGzip.cpp`.
+| `pipe_utils.hpp`             | Header file containing the definition of an array of pipes. 
```

Comment:
Please add a note to all README files to indicate that pipe_utils is in a different directory, and where it can be found.  I'll just comment on this one.

Reply:
Added the following to the README.md lines:
|`pipe_utils.hpp`                | Header file containing the definition of an array of pipes and a pipe duplicator. This header can be found in the DirectProgramming/DPC++FPGA/include/ directory of this repository.